<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_OpenBluetooth">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>打开蓝牙</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CloseBluetooth">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>110</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭蓝牙</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_BluetoothName">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>391</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StopScan">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>40</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>停止扫描</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_BluetoothScan">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>开始扫描</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_BluetoothDevice">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>451</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_BluetoothInfiShow">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>107</width>
      <height>107</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_UUID">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>451</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_SendData">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>450</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ConnectDev">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>连接蓝牙</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
